core/domain/stats_services.py

```diff
@@ -543,7 +536,7 @@ def get_exp_issues_from_model(exp_issues_model):
     """
     unresolved_issues = []
     for unresolved_issue_dict in exp_issues_model.unresolved_issues:
-        _migrate_to_latest_issue_schema(copy.deepcopy(unresolved_issue_dict))
```

Comment:
Why did you remove this? Doesn't seem code-coverage related.

Reply:
Using deepcopy was not updating the unresolved_issue_dict.

Reply:
Try fetching the new dict before testing it, then.

Reply:
Resolved offline — This code is buggy, we should be using the deepcopy.